src/EditorFeatures/Core/Implementation/AutomaticCompletion/AbstractAutomaticLineEnderCommandHandler.cs

```diff
@@ -58,6 +63,22 @@ public VSCommanding.CommandState GetCommandState(AutomaticLineEnderCommandArgs a
 
         public void ExecuteCommand(AutomaticLineEnderCommandArgs args, Action nextHandler, CommandExecutionContext context)
         {
+            // Completion will only be active here if this command wasn't handled by the completion controller itself.
+            if (_asyncCompletionBroker.IsCompletionActive(args.TextView))
+            {
+                var session = _asyncCompletionBroker.GetSession(args.TextView);
+                var computedItems = session.GetComputedItems(context.OperationContext.UserCancellationToken);
+                var softSelection = computedItems.SuggestionItemSelected || computedItems.UsesSoftSelection;
+                var behavior = session.Commit('\n', context.OperationContext.UserCancellationToken);
```

Comment:
If we do go with taking this workaround, I presume the code here should have a comment linking to the tracking bug to track the proper fix?